libs/ledger-live-common/src/families/bitcoin/logic.ts

```diff
@@ -216,7 +216,8 @@ export const mapTxToOperations = (
       : tx.outputs.map(o => o.output_index).reduce((p, c) => (p > c ? p : c));
 
   for (const output of tx.outputs) {
-    if (output.address) {
+    // ledger explorer returns "unknown" as recipient for OP_RETURN outputs, we don't want to display it in our UI
+    if (output.address && !output.address.includes("unknown")) {
```

Comment:
Is this the only way to identify an OP_RETURN output?
Best way would be from blockchain data instead of some specific response from Ledger explorer, but otherwise it's fine.

Reply:
We can also check the "script_hex" field, but it is more complicated. Also, the OP_RETURN output is not the only "unknown" address output, (bitcoin spec has some more complicated output with "unknown" address) 
but we should filter out all those output as we can't support such operations in LL so far. (i.e. we can broadcast a transaction with OP_RETURN output but we can't display such outputs without address in LLD/LLM )